index.html

```diff
@@ -3070,25 +3087,121 @@ <h2>
             <dd>
 If the resolution is successful, this MUST be a <a
 href="metadata-structure">metadata structure</a>. This structure contains
-metadata about the <a>DID document</a> contained in the
+metadata about the <a>DID document</a> contained in the <code>did-document</code> or
 <code>did-document-stream</code>. This metadata typically does not change
 between invocations of the <code>resolve</code> function unless the <a>DID
 document</a> changes, as it represents data about the <a>DID document</a>. If
 the resolution is unsuccessful, this output MUST be an empty <a
 href="metadata-structure">metadata structure</a>.
+Properties defined by this specification are in <a href="#did-document-metadata-properties"></a>.
             </dd>
         </dl>
 
         <p>
-<a>DID resolver</a> implementations MUST NOT alter the signature of this
-function in any way. <a>DID resolver</a> implementations MAY map the
-<code>resolve</code> function to a method-specific internal function to perform
+<a>DID resolver</a> implementations MUST NOT alter the signature of these
+functions in any way. <a>DID resolver</a> implementations MAY map the
+<code>resolve</code> and <code>resolveStream</code> functions to a method-specific internal function to perform
 the actual <a>DID resolution</a> process. <a>DID resolver</a> implementations
 MAY implement  and expose additional
 functions with different signatures in addition to the <code>resolve</code>
 function specified here.
         </p>
 
+        <section>
+            <h3>
+DID Resolution Input Metadata Properties
+            </h3>
+            
+            <p>
+The possible properties within this structure and their possible values are defined by [[DID-SPEC-REGISTRIES]].
+This specification defines the following common properties.
+            </p>
+            
+            <dl>
+                <dt>
+accept
+                </dt>
+                <dd>
+The MIME type of the caller's preferred <a href="#representations">representation</a> of the <a>DID document</a>. The <a>DID resolver</a> implementation
+MAY use this value to determine the representation contained in the returned <code>did-document-stream</code> if such
```

Comment:
```suggestion
SHOULD use this value to determine the representation contained in the returned <code>did-document-stream</code> if such
```

Reply:
We may want to tighten this up a bit more and say that it MUST be used to determine the representation if used with `resolveStream` and MUST be ignored if used with `resolve`. That would better fall in line with other statements made elsewhere. Or -- we could just say what the format is here and leave those other normative statements in one place with their respective functions.

Reply:
I agree with tightening this and would probably also prefer MUST. We can add an error code `unsupported-representation` if the representation is not supported (this would be comparable to an HTTP 406 error).

Reply:
"SHOULD" is fine but "MUST" is too far here.

Reply:
I'll accept the "SHOULD" suggestion for now, we can later raise a separate issue/PR to change it if desired.